infra-as-code/bicep/modules/privateDnsZones/privateDnsZones.bicep

```diff
@@ -83,6 +83,10 @@ param parTags object = {}
 @sys.description('Resource ID of VNet for Private DNS Zone VNet Links.')
 param parVirtualNetworkIdToLink string = ''
 
+@sys.description('Resource ID of VNet for Failover Private DNS Zone VNet Links.')
+param parVirtualNetworkIdToLinkFailover string = ''
```

Comment:
do we consider renaming `Failover` to `Alternate`, as its not really a failover as such?

Would require updates to names in other places :) But think its the right thing to do

Reply:
In the documentation of azure private dns resolver for example a second link is documented as failover.

Reply:
Can you share a link?

Reply:
https://learn.microsoft.com/en-us/azure/dns/private-resolver-reliability  @jtracey93